docs/design/autoware-interfaces/components/localization.md

```diff
@@ -0,0 +1,67 @@
+# Localization
+
+![Node diagram](images/Localization-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### PointCloud Map
+
+Environment map created with point cloud, Published by the Map Server.
+
+- sensor_msgs::msg::PointCloud2
+
+### Initial Pose
+
+Start pose of ego, Published by the User Interface.
+
+- geometry_msgs::msg::PoseCovarianceStamped
+
+### 3D-LiDAR Scanning
+
+LiDAR scanning for NDT matching. Published by the LiDAR sensor.
+
+- sensor_msgs::msg::PointCloud2
+
+### GNSS
+
+Current Geographic coordinate of the ego. Published by GNSS sensor.
+
+- sensor_msgs::msg::NavSatFix
```

Comment:
```suggestion
- sensor_msgs::msg::NavSatFix
- autoware_sensing_msgs::msg::GnssInsOrientationStamped
```
Incidentally, if we also need velocity and acceleration, we can use
TwistWithCovarianceStamped and AccelWithCovarianceStamped.
https://github.com/autowarefoundation/autoware_msgs/tree/main/autoware_sensing_msgs